front/src/applications/stdcm/components/SimulationReportSheet.tsx

```diff
@@ -32,6 +34,16 @@ const SimulationReportSheet = ({
     path_number2: 'n°YYYYYY',
   };
 
+  const getArrivalTime = (step: PathStep, isFirstStep: boolean, isLastStep: boolean) => {
```

Comment:
Use this syntax. It's more easier to read
```suggestion
  const getArrivalTime = (step: PathStep, { isFirstStep, isLastStep }: { isFirstStep?: boolean; isLastStep?: boolean }) => {
```

Reply:
done